package/MDAnalysis/analysis/pcz.py

```diff
@@ -0,0 +1,481 @@
+    ltarg = reference - targ_com
+    R, rms = rotation_matrix(ltarg, lmob, weights = weights)
+    return rms
+
+class Fasu:
+    def __init__(self, universe, atom_selection='name *', start=None, stop=None, step=None):
+        """
+        A "Fasu" is a convenience wrapper round an MDAnalysis universe,
+        providing slicing and pseudo-direct access methods for any 
+        trajectory format.
+
+        :Arguments:
+          *universe*
+            :class:`MDAnalysis.Universe` object
+
+          *atom_selection* 
+            a valid selection string for 
+            :meth:`MDAnalysis.core.AtomGroup.AtomGroup.select_atoms`
+
+          *start*
+            int, the index (zero-based) of the first frame in the trajectory
+            to include in the PC analysis
+
+          *stop*
+            int, the index (zero-based) of the last frame in the trajectory 
+            to include in the PC analysis
+
+          *step*
+            int, the increment between frames to be included in the analysis
+        """
+        self._u = universe
+        self._sel = self._u.select_atoms(atom_selection)
+        self.natoms = len(self._sel)
+        if start is None:
+            self._start = 0
+        else:
+            self._start = start
+        if stop is None:
+            self._stop = self._u.trajectory.n_frames
+        else: 
+            self._stop = stop
+        if step is None:
+            self._step = 1
+        else:
+            self._step = step
+
+        self.numframes = abs(self._stop - self._start)/self._step
+
+    def coords(self, frame_number, reference_structure = None):
+        """
+        Returns a [natoms, 3] numpy array of the selected coordinates at the
+        chosen frame (zero-based). The coordinates are least-squares
+        fitted to reference_structure, if this is specified.
+
+        :Arguments:
+          *frame_number*
+            int, the index (zero-based) of the desired trajectory frame
+
+          *reference_structure*
+            :class:`numpy.ndarray` containing the coordinates of the 
+            reference structure
+
+        :Returns:
+          :class:`numpy.ndarray` containing the coordinates of the selected 
+          snapshot
+        """
+        f_real = self._start + frame_number * self._step
+
+        if f_real < (self._u.trajectory.frame - 1):
+            self._u.trajectory.rewind()
+        while (self._u.trajectory.frame ) < f_real:
+            try:
+                self._u.trajectory.next()
+            except EOFError:
+                print 'End of file error trying to access frame ', frame
+                raise EOFError
+
+        crds = self._sel.ts._pos
+        if reference_structure is not None:
+            crds = lsqfit(crds, reference_structure)
+        return crds
+
+class Pca:
+    def __init__(self, universe, atom_selection=None, start=None, stop=None,
+    step=None, reference_coordinates=None, fitting_tolerance = 0.01,
```

Comment:
It should line up on the opening brace

```python
    def __init__(self, universe, atom_selection=None, start=None, stop=None,
                        step=None, reference_coordinates=None, fitting_tolerance = 0.01,
                        max_iterations = 10, captured_variance = None ):
```

To keep you from manual editing everything you can use [yapf](https://github.com/google/yapf) to autoformat the code